### ISSUE TITLE
Update readme

### ISSUE THREAD

**Issue by MadScrewdriver** [jcasoft/Web-Chat-Client-for-Mycroft#3]
To get it working you have to:

```
git clone https://github.com/jcasoft/Web-Chat-Client-for-Mycroft.git ~/Downloads/chat
cd ~/mycroft-core/mycroft/messagebus/service 
mv ~/Downloads/chat/* .
nano main_1.py 
``` 
and remove "kill," from line 31

```
mv __main__.py original_main.py
mv main_1.py __main__.py
```

install all the dependencis with `pip install --user <pacage>`
and you are all set

**Comment by defaultusername1152** [comment]
What dependencies?

**Comment by MadScrewdriver** [comment]
My bad that I didn't explain it more carefully in May. Today I don't remember a thing. 
Anyway.
Most likely dependencies that are needed for file main_1.py to run. If you look into the file. you can see that it has dependencies such as " tornado" pacage

**Comment by MadScrewdriver** [comment]
Yep, it looks like tornado is the only dependency that needs to be installed.
Run `pip install --user tornado`
